packages/ui-admin/src/app/routes/NetPromotingScore/NPS.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode } from 'react'
+import NPSScale from './components/NPSScale'
+import styles from './style.scss'
+interface Props {
+  animated?: boolean
+  dismissed?: boolean
+  score?: number | null
+  question?: string
+  scaleWorstLabel?: string
+  scaleBestLabel?: string
+  onSubmit?: (score: number) => void
+  onDismissed?: () => void
+  children?: ReactNode
```

Comment:
I modified a props to be able to add children component inside the main component